### PR TITLE
fix (core) : TransportMgr Load up time

### PR DESCRIPTION
**Pull request by acidmanifesto**
This may fix the brief pause on the world startup when it comes to Loading Transports....
This seems to be working for me on windows 10 pro. WIll need other OS build testing
This was something thats been annoying the hell out of me and malcrom.
I under stand load times will vary, But this is the first time i gotten a load time under 20ish seconds on my system

https://user-images.githubusercontent.com/16887899/151405258-e7907746-bb71-4549-99c3-22204faf7e83.mp4

image

## Changes Proposed:
-  Cleaned up the  void TransportMgr::SpawnContinentTransports() 

## Issues Addressed:

- Closes 

## Tests Performed:

- Build
- Performs

## How to Test the Changes:

1. This should prevent the delay pause on loading transports on load up
2. Only tested on Windows 10 Pro Local Server.
3.

## Known Issues and TODO List:

- [ ]
- [ ]

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

File: src/server/game/Maps/TransportMgr.cpp [hunk 1]
```diff
@@ -68,13 +68,13 @@ void TransportMgr::LoadTransportTemplates()
         GameObjectTemplate const* goInfo = sObjectMgr->GetGameObjectTemplate(entry);
         if (goInfo == nullptr)
         {
-            LOG_ERROR("entities.transport", "Transport {} has no associated GameObjectTemplate from `gameobject_template` , skipped.", entry);
+            LOG_ERROR("entities.transport", "Transport %u has no associated GameObjectTemplate from `gameobject_template` , skipped.", entry);
```
Comment (Kitzunu) [inline src/server/game/Maps/TransportMgr.cpp:71]: Recert these changes

**Review by Kitzunu — DISMISSED** [review]

**Review by malcrom — APPROVED** [review]

File: src/server/game/Maps/TransportMgr.cpp [hunk 2]
```diff
@@ -423,13 +423,13 @@ void TransportMgr::SpawnContinentTransports()
     if (_transportTemplates.empty())
         return;
 
-    uint32 oldMSTime, count = 0;
-    oldMSTime = getMSTime();
-    QueryResult result;
+    uint32 count = 0;
+    uint32 oldMSTime = getMSTime();
+    QueryResult result = WorldDatabase.Query("SELECT guid, entry FROM transports");
```
Comment (Goatform) [inline src/server/game/Maps/TransportMgr.cpp:428]: Why is `result` exposed if there's `CONFIG_ENABLE_CONTINENT_TRANSPORT`, on other words, why did you move it outside if there's no use for it code below in case `CONFIG_ENABLE_CONTINENT_TRANSPORT` false (is disabled) ?

Comment (acidmanifesto) [reply src/server/game/Maps/TransportMgr.cpp:428]: Result was always defined with the wdb query to begin with even before this pr. Please test and look over it in debugger yourself before and after the pr adjustments. Both configs were tested and are still functional with no further issues.
